lib/query-builder-filters.js

```diff
@@ -228,7 +228,7 @@ class QueryBuilderFilters {
 		let value = filter.value || filter;
 
 		if(QueryBuilderFields._isFlagField(fieldName))
-			value = value ? QueryBuilderFields._getFlagData(fieldName).value : 0;
+			value = ['string', 'number', 'boolean'].includes(typeof value) ? value : QueryBuilderFields._getFlagData(fieldName).value;
```

Comment:
Che, `_getFlagData()` recibe un string. Y nunca va a llegarle, porque no entra al else.. Puede ser que no tenga sentido o hay algo que no estoy viendo? :thinking:

Reply:
De alguna manera los test entran, asique no sé, no estoy entendiendo algo creo..

Reply:
@gastonpereyra Ahí estuvimos viendo con Manu. Lo que tiene que pasar en realidad, es que si te mandan algún valor que deba ser considerado como `false` (`0, false, '0', 'false') se debe filtrar con el valor `0`. En cualquier otro caso, se debe filtrar con el valor de la flag.

Con este cambio debería estar OK:

```suggestion
			value = !value || value === '0' || value === 'false' ? 0 : QueryBuilderFields._getFlagData(fieldName).value;
```